### DISCUSSION TITLE
How to select a different database on ArcadeDB website?

### DISCUSSION THREAD

**Discussion by ZM-TSS** [ArcadeData/arcadedb#967] [General]
I cannot see my databases on ArcadeDB Website. 
But I can see them both on the CLI. 
What am I missing on setting-up or running-on ArcadeDB?

Setup with our [Getting-Started](https://arcadedb.com/) Guide

```bash
> docker run --rm -p 2480:2480 -p 2424:2424
           -e JAVA_OPTS="-Darcadedb.server.rootPassword=playwithdata -Darcadedb.server.defaultDatabases=Imported[root]{import:https://github.com/ArcadeData/arcadedb-datasets/raw/main/orientdb/OpenBeer.gz}"
           arcadedata/arcadedb:latest
```
Create 2nd food DB with their [ArcadeDB-Doc's JSON-Importer](https://docs.arcadedb.com/#JSON-Importer)
```bash
$ create database food 
$ import database file://home/arcadedb/Downloads/FoodData_Central_foundation_food_json_2022-10-28.zip with mapping = {"FoundationFoods": [{"@cat": "v","@type": "<foodClass>","foodNutrients": [{"@cat": "e","@type": "HAS_NUTRIENT","@in": "nutrient","@cardinality": "no-duplicates","nutrient": {"@cat": "v","@type": "Nutrient","@id": "id","@idType": "long","@strategy": "merge"},"foodNutrientDerivation": "@ignore"}],"inputFoods": [{"@cat": "e","@type": "INPUT","@in": "inputFood","@cardinality": "no-duplicates","inputFood": {"@cat": "v","@type": "<foodClass>","@id": "fdcId","@idType": "long","@strategy": "merge","foodCategory": {"@cat": "e","@type": "HAS_CATEGORY","@cardinality": "no-duplicates","@in": {"@cat": "v","@type": "FoodCategory","@id": "id","@idType": "long","@strategy": "merge"}}}}]}]}

```
From that, With a slight substitution because its in a Docker-container instead of a local-instance; To this

```bash
$ docker cp FoodData_Central_foundation_food_json_2022-10-28.zip 
$ winpty docker exec -it arcadeDB_container sh 
# bin/console.sh 
> create database food 
> import database file://home/arcadedb/Downloads/FoodData_Central_foundation_food_json_2022-10-28.zip with mapping = {"FoundationFoods": [{"@cat": "v","@type": "<foodClass>","foodNutrients": [{"@cat": "e","@type": "HAS_NUTRIENT","@in": "nutrient","@cardinality": "no-duplicates","nutrient": {"@cat": "v","@type": "Nutrient","@id": "id","@idType": "long","@strategy": "merge"},"foodNutrientDerivation": "@ignore"}],"inputFoods": [{"@cat": "e","@type": "INPUT","@in": "inputFood","@cardinality": "no-duplicates","inputFood": {"@cat": "v","@type": "<foodClass>","@id": "fdcId","@idType": "long","@strategy": "merge","foodCategory": {"@cat": "e","@type": "HAS_CATEGORY","@cardinality": "no-duplicates","@in": {"@cat": "v","@type": "FoodCategory","@id": "id","@idType": "long","@strategy": "merge"}}}}]}]}
> list databases 

Databases:
- Imported 
- food 
```

Both the CLI and the website know about food DB: As seen by the list databases on the CLI, and As seen by the create food DB response on the website Cannot execute command Database 'food' already exists. But I cannot see-or-select on the website dropdown options.

Why can I not use the food DB on the website?

I've tried:
1. Restarting and recreating the Docker container
2. Restarting Docker and the host-computer
3. Refreshing, Re-logging-in, and incognito-mode-ing the ArcadeDB website
I was expecting to see the DB show up on the Website selection options.

I see everything but the website allowing me to use the other DB from the dropdown selection.

**Comment by ZM-TSS** [comment]
- [discord.com/channels/883057780553052180/883057780553052184/…](https://discord.com/channels/883057780553052180/883057780553052184/1088810608167759897) 
- [github.com/ArcadeData/arcadedb/discussions/…](https://github.com/ArcadeData/arcadedb/discussions/967#discussion-4996310)
- [github.com/ArcadeData/arcadedb/issues/645#issue-1466408318](https://github.com/ArcadeData/arcadedb/issues/645#issue-1466408318)

**Comment by IC088** [comment]
Hi I am having a similar problem where i can see my database in the Browser UI but when i try to list out from the container console, i am unable to see.

**Reply by gramian** [reply]
Hi,
if you want to access a database of a running server from the console in the same container then you have to make a **remote** connection from the console. A local connection will not work due to transaction guarantees AFAIK, ie two independent processes writing to the same files.
